### ISSUE TITLE
Currency history graph

### ISSUE THREAD

**Issue by Leoguy77** [currency-cop/currency-cop#64]
hey guys!

at first, thanks for your great tool! i really love it.

but im missing a feature i would really like to see in a future release:

a currency history.
I think about it like a chart of when you had how much currency. it would be a really cool feature.
there is already a history you implemented, but it only shows the last few minutes, but it would need to save it to a file and show all the entrys as a graph.

Hope you can implement something like this :)

Leoguy77

**Comment by ADRDev** [comment]
I'm working on this for my own use (and to learn a bit of electron/react).

![image](https://user-images.githubusercontent.com/11244631/41942067-1188f7a8-796c-11e8-98e7-2414d052e315.png)

I'm using react-vis component for graph.

@nijikokun Would you like me to continue with this and submit a pull request, or do you have something else in mind?

**Comment by nijikokun** [comment]
@ADRDev I'd love a PR, however, it would be better if it used highchart and was a formalized aggregation plot over a month (30 day bucket) / day (24 hour buckets) rather than using the history list we have now

**Comment by ADRDev** [comment]
@nijikokun Yes, aggregating the data was going to be my next step. I'll switch to highchart and proceed along those lines.
